fineract-provider/src/main/java/org/apache/fineract/organisation/workingdays/domain/WorkingDays.java

```diff
@@ -74,6 +78,8 @@ public Boolean getExtendTermForDailyRepayments(){
         return this.extendTermForDailyRepayments;
     }
 
+    public Boolean getExtendTermForRepaymentsOnHolidays() { return this.extendTermForRepaymentsOnHolidays; }
+
     public Map<String, Object> update(final JsonCommand command) {
```

Comment:
No code changes that would set this variable is written..... no one can change this attribute